app-text/crm114/metadata.xml

```diff
@@ -1,19 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="project">
-    <email>shell-tools@gentoo.org</email>
-    <name>Gentoo Shell Tools Project</name>
-  </maintainer>
-  <use>
-    <flag name="mew">Add support for using the mewdecode mime decoder
-    (<pkg>app-emacs/mew</pkg>)</flag>
-    <flag name="mimencode">Add support for using the mimencode mime
-    (<pkg>net-mail/metamail</pkg>)</flag>
-    <flag name="normalizemime">Add support for using the normalizemime
-    (<pkg>mail-filter/normalizemime</pkg>)</flag>
-  </use>
-  <upstream>
-    <remote-id type="sourceforge">crm114</remote-id>
-  </upstream>
+	<maintainer type="project">
+		<email>shell-tools@gentoo.org</email>
+		<name>Gentoo Shell Tools Project</name>
+	</maintainer>
+	<use>
+		<flag name="mew">Add support for using the mewdecode mime decoder (<pkg>app-emacs/mew</pkg>)</flag>
+    	<flag name="mimencode">Add support for using the mimencode mime (<pkg>net-mail/metamail</pkg>)</flag>
```

Comment:
indent. ever tried `XMLLINT_INDENT="	" xmllint --format` (that's a tab character)?

Reply:
neat, thanks :)